web/client/plugins/SearchServicesConfig.jsx

```diff
@@ -0,0 +1,211 @@
+/**
+* Copyright 2017, GeoSolutions Sas.
+* All rights reserved.
+*
+* This source code is licensed under the BSD-style license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+const React = require('react');
+const {connect} = require('react-redux');
+const assign = require('object-assign');
+const {Glyphicon, Button} = require('react-bootstrap');
+const ConfirmButton = require('../components/buttons/ConfirmButton');
+const Dialog = require('../components//misc/Dialog');
+const Portal = require('../components/misc/Portal');
+const Message = require('./locale/Message');
+const {isEqual} = require('lodash');
+const {toggleControl} = require('../actions/controls');
+const {setSearchConfigProp, updateService, restServiceConfig} = require('../actions/searchconfig');
+
+require('../components/mapcontrols/searchservicesconfig/SearchServices.css');
+
+const SearchServicesButton = require('./searchservicesconfig/ToggleButton');
+const ServiceList = require('../components/mapcontrols/searchservicesconfig/ServicesList.jsx');
+const WFSServiceProps = require('../components/mapcontrols/searchservicesconfig/WFSServiceProps.jsx');
+const ResultsProps = require('../components/mapcontrols/searchservicesconfig/ResultsProps.jsx');
+const WFSOptionalProps = require('../components/mapcontrols/searchservicesconfig/WFSOptionalProps.jsx');
+
+
+const SearchServicesConfigPanel = React.createClass({
```

Comment:
Document the plugin. see http://dev.mapstore2.geo-solutions.it/mapstore/docs/building-and-developing